src/components/ConditionalFilter/API/initialState/orders/InitialOrderState.ts

```diff
@@ -44,16 +46,16 @@ export class InitialOrderStateResponse implements InitialOrderState {
       return token.value;
     }
 
-    return this.getEntryByName(token.name).filter(({ slug }) => {
-      if (isTextInput(token.name)) {
-        return true;
-      }
+    const entry = this.getEntryByName(token.name);
 
-      return slug && token.value.includes(slug);
-    });
+    if (isTextInput(token.name)) {
+      return entry;
+    }
+
+    return (entry as ItemOption[]).filter(({ slug }) => slug && token.value.includes(slug));
```

Comment:
suggestion: entry can be array of string from what I see below `ItemOption[] | string | string[]` and I don't see you handle this case